Doc/library/turtle.rst

```diff
@@ -2340,15 +2342,15 @@ The built in configuration would correspond to the following turtle.cfg::
 
 Short explanation of selected entries:
 
-- The first four lines correspond to the arguments of the :meth:`Screen.setup`
+- The first four lines correspond to the arguments of the :meth:`turtle.setup`
```

Comment:
For these, where the mentions contain the additional information as to the specific class these methods belong to, you could retain that while still fixing the problem by using an explicit title. Furthermore, as documented these are module-level functions, not class-level methods, and using a redundant explicit `turtle` here implies that `turtle` is the class (as I initially thought), not just the (redundant) module name. So, I would suggest:

```suggestion
- The first four lines correspond to the arguments of the :func:`Screen.setup <setup>`
```

And the same with the others.